src/app/clusters/content-launch-server/content-launch-server.cpp

```diff
@@ -154,6 +146,7 @@ class ContentLauncherAttrAccess : public app::AttributeAccessInterface
 private:
     CHIP_ERROR ReadAcceptHeaderAttribute(app::AttributeValueEncoder & aEncoder, Delegate * delegate);
     CHIP_ERROR ReadSupportedStreamingProtocolsAttribute(app::AttributeValueEncoder & aEncoder, Delegate * delegate);
+    CHIP_ERROR ReadFeatureFlagAttribute(EndpointId endpoint, app::AttributeValueEncoder & aEncoder, Delegate * delegate);
```

Comment:
Will we have to do this for commandlist & attributelist as well?

Reply:
no, this part seems to work already based upon the code in AppImpl which creates the dynamic endpoint. There is just not a way to set the feature flag using those methods

Reply:
I see. But that code currently assumes that every content app supports the same set of clusters/commands/attributes. We can modify that later.